pages/osx/autoraise.md

```diff
@@ -0,0 +1,8 @@
+# AutoRaise
+
+> Add Focus-Follows-Mouse to MacOS.
```

Comment:
From https://github.com/sbmpost/AutoRaise.

The previous description is kind of hard to understand in my opinion.
```suggestion
> Automatically raise and focus on a window when hovering over it with the mouse.
```